Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stretch
+
+LABEL maintainer="tworr@usgs.gov"
+
+# Run updates and install curl
+RUN apt-get update && \
+      apt-get upgrade -y && \
+      apt-get install curl -y && \
+      apt-get purge -y --auto-remove && \
+      apt-get clean
+      
+# Enable the NodeSource repository and install the latest nodejs
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
+      apt-get install nodejs -y
+
+# Create temp directory for building viz app
+RUN mkdir -p /tmp/wbeep-viz-app
+
+# Copy source code.  Note that we need to exclude the mbtiles
```

Comment:
Not being all that familiar with Docker, take what I say with a grain of salt. But, I am 99.9% sure that the 'mbtiles' folder in the repository is a legacy item that should have been removed in previous pull requests that I made, but I missed it. I am not seeing where in the code here where it is being excluded, but since the 'mbtiles' folder should just be deleted anyway, there would be no need to exclude it. :)

Reply:
Oh, I see where the 'mbtiles' are ignored. Up in the '.dockerignore.' Who would have guessed?